stetho/src/main/java/com/facebook/stetho/inspector/elements/android/FragmentDescriptor.java

```diff
@@ -64,6 +68,8 @@ protected int onGetChildCount(Object element) {
 
   @Override
   protected Object onGetChildAt(Object element, int index) {
+    verifyAccess();
```

Comment:
I think this is way overzealous.  I believe you should enforce this only at the root node of the tree and then just document that you must not leak or manipulate children views manually without also taking care of thread safety and ensuring you don't leak.  You're making one of the ways this is dangerous safe, but leaving all the other ways around.